### PR TITLE
Make server disambiguation in Github/Bitbucket italic

### PR DESCRIPTION
**Pull request by NicuPascu**
# Description

See [JENKINS-46187](https://issues.jenkins-ci.org/browse/JENKINS-46187).

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

**Comment by NicuPascu** [comment]
italic-server

**Comment by i386** [comment]
LGTM 👍

**Comment by jlong** [comment]
screen shot 2017-08-15 at 10 24 05 am

@i386 @NicuPascu Is there a way that we could put the GitHub options and the BitBucket options side by side. The current arrangement is hard to parse.

**Comment by i386** [comment]
@jlong I am not a fan of that design, as it gives too much weight to the Git option when they are all equal choices.

**Comment by jlong** [comment]
@i386 Looking back at an earlier version of this, I believe Brody had it at the bottom:

screen shot 2017-08-15 at 6 59 40 pm

How do you feel about this option?

**Comment by i386** [comment]
@jlong yes, though I'd change the order so it was more alphabetical
# Bitbucket
# GitHub
# Git

**Comment by jlong** [comment]
WFM :)
